TileSystem/Interfaces/Management/IManageEntities.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System;
+
+namespace TileSystem.Interfaces.Management
+{
+	/// <summary>
+	/// Used by the tiles to manage the entities being moved
+	/// between them
+	/// </summary>
+	public interface IManageEntities
```

Comment:
Will keep these separate so we can have creation and management separate for all objects in the system.